apps/ui/src/views/AboutView.vue

```diff
@@ -12,10 +12,91 @@
         <iframe src="https://www.youtube.com/embed/MfzmM3EFqIY?si=Q0ovRqJmr46-NN7z" allowfullscreen class="video-player" />
       </div>
     </div>
+    <div class="team mt-10">
+      <h1>Our Team</h1>
+      <div v-if="queryData.loading.value" class="loading mb-10">
+        <v-progress-circular color="red-darken-1" size="80" indeterminate>
+        </v-progress-circular>
+        <p style="color: white; text-align: center">Loading Members...</p>
+      </div>
+      <div v-else class="membership">
+        <h2 v-if="officers.length" class="ml-5">Leadership</h2>
+          <div class="members mt-2 ml-5">
+            <div v-for="officer in officers" :key="officer.id">
+              <RouterLink class="member-container" :to="personURL + officer.person?.id">
+                <v-avatar :rounded="officer.person?.profilePicture?.path ? 'default' : 0" size="150"
+                          :image="officer.person?.profilePicture?.path ?? rpitvLogo" />
+                <h3>{{ officer.person?.name }}</h3>
+                <h4 class="role">{{ officer.role?.name }}</h4>
+              </RouterLink>
+            </div>
+          </div>
+        <h2 v-if="members.length" class="ml-5">Members</h2>
+        <div class="members mt-2 ml-5">
+          <div v-for="member in members" :key="member.id">
+            <RouterLink class="member-container" :to="personURL + member.person?.id">
+              <v-avatar :rounded="member.person?.profilePicture?.path ? 'default' : 0" size="150"
+                        :image="member.person?.profilePicture?.path ?? rpitvLogo" />
+              <h3>{{ member.person?.name }}</h3>
+              <h4 class="role">{{ member.role?.name }}</h4>
```

Comment:
Consider using another element, e.g. `div`, with styled text size to avoid screen reader/accessibility issues:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Accounted for.